src/adcm_client/objects.py

```diff
@@ -545,10 +545,40 @@ def bind(self, target):
         else:
             raise NotImplementedError
 
-    def bind_list(self, paging=None):
+    def _bind_list_old(self, paging=None):
         """Provide endpoint to bind/list"""
         return self._subcall("bind", "list")
 
+    @legacy_server_implementaion(_bind_old, '2022.02.1.00')
+    def bind(self, target) -> "ClusterBind":
+        """Create new ClusterBind object and return it"""
+        if isinstance(target, Cluster):
+            data = self._subcall("bind", "create", export_cluster_id=target.cluster_id)
+        elif isinstance(target, Service):
+            data = self._subcall(
+                "bind",
+                "create",
+                export_cluster_id=target.cluster_id,
+                export_service_id=target.service_id,
+            )
+        return ClusterBind(self._api, id=data['id'])
+
+    @min_server_version('2022.02.1.00')
+    def bind_delete(self, bind: "ClusterBind"):
+        """Delete service from cluster"""
+        with allure_step(f"Remove bind {bind.id} from cluster {self.name}"):
+            self._subcall("bind", "delete", bind_id=bind.id)
+
+    @min_server_version('2022.02.1.00')
+    def get_bind(self, **kwargs):
```

Comment:
I think this method is redundant. `Bind(cluster_id=1, service_id=1, id=1)` for service or `Bind(cluster_id=1, id=1)` for cluster  will suffice.